include/sound/hdaudio_ext.h

```diff
@@ -117,17 +117,17 @@ struct hdac_ext_link {
 	struct list_head list;
 };
 
-int snd_hdac_ext_bus_link_power_up(struct hdac_ext_link *link);
```

Comment:
just a minor typo in commit message "all the file" -> "all the files"

Reply:
removed.